OperationalExcellenceAccNetworkingSample/Deployment/accNetwork.json

```diff
@@ -0,0 +1,495 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {
+        "accNetworkVnetName": "accNetwork-vnet",
+        "accNtwkStgAccountName": "accnetworkdstg"
```

Comment:
Ran into an error when deploying the template because the name of the storage account was already in use. I suggest you dynamically create a name here that is unique to the subscription and resource group name.

Reply:
Fixed, made the storage account name dynamic